prod-refractr.yml

```diff
@@ -832,12 +824,115 @@ refracts:
 - www.mozilla.org/: status.mozilla.org
   status: 302
 
+# https://mozilla-hub.atlassian.net/browse/SE-2450
 - dsts:
-  - if: '$request_uri ~ "(?i)^/universityambassadors/?$"'
-    ^: 'campus.mozilla.community/?redirect_source=firefox-com'
+    - /: 'www.mozilla.org/firefox/new/?redirect_source=firefox-com'
```

Comment:
Just to confirm my understanding, the default situation is strictly matching www.firefox.com is redirected to the above domain - not any www.firefox.com path that doesn't match the following paths?

Asked another way, if someone requests www.firefox.com/blah/de/do, it should 404, not go to the above redirect?